### PR TITLE
feat(controller): set up integration tests

### PR DESCRIPTION
**Pull request by marouaneaba**
Set up integration tests for stock.

### CODE REVIEW

**Comment by marouaneaba** [comment]
I am sorry I fork this projet to understand the architecture and design, I did not pay attention that I am pointing on this repo.

**Comment by hypr2771** [comment]
@marouaneaba all good, good luck =)

File: pom.xml [hunk 1]
```diff
@@ -28,6 +30,7 @@
 
   <properties>
     <java.version>14</java.version>
+    <junitVersion>4.13.2</junitVersion>
```
Comment (hypr2771) [inline pom.xml:33]: Be careful to remain coherent with other naming convention.
Here it seems we were using _dot notation_.

File: common/src/main/java/com/example/demo/ShoeApiEndpoints.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package com.example.demo;
+
+/**
+ * <p>
```
Comment (hypr2771) [inline common/src/main/java/com/example/demo/ShoeApiEndpoints.java:4]: Is this standard to break lines using a non-self-closing `<p>`?
Isn't `<br />` a more standard practice?

File: common/src/main/java/com/example/demo/ShoeApiEndpoints.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+ * <p>
+ * Centralise l'ensemble des endpoints disponibles pour les apis RESTS.
```
Comment (hypr2771) [inline common/src/main/java/com/example/demo/ShoeApiEndpoints.java:6]: I would suggest being careful with casing.
_API_ is an acronym, so is _REST_, hence it has no plural, and _RESTS_ is not an acronym.

**Review by hypr2771 — COMMENTED** [review]
Felt like giving you a feedback.

File: pom.xml (hunk 1 above)
Comment (marouaneaba) [reply pom.xml:33]: Noted, thank you for the remarks 😊.

File: common/src/main/java/com/example/demo/ShoeApiEndpoints.java (hunk 2 above)
Comment (marouaneaba) [reply common/src/main/java/com/example/demo/ShoeApiEndpoints.java:4]: Noted

**Pull request closed without merging**